IBM/carbon-addons-iot-react
# style/alignment updates: PageTitleBar, Table (pagination)

**Summary**

- Fixes to alignment and styling for PageTitleBar and Table (pagination)
- Added `extraContent` prop in PageTitleBar, added deprecation comment for `rightContent`

**Change List (commits, features, bugs, etc)**

* PageTitleBar styling / `extraContent` alignment

![image](https://user-images.githubusercontent.com/2175647/67226798-719d8400-f3fb-11e9-8e3b-5742151f8e67.png)

* Fixed alignment issue in pagination select indicator

![image](https://user-images.githubusercontent.com/2175647/67226635-05bb1b80-f3fb-11e9-8436-1d08e00c4e84.png)

**Related Issues**



- Closes IBM/carbon-addons-iot-react#618

**Acceptance Test (how to verify the PR)**

- Verify all permutations of the PageTitleBar story
- Verify a Table story with pagination
